sass/layout/_container.sass

```diff
@@ -0,0 +1,28 @@
+body
+    display: block
+    background-color: $brand-secondary
+    font-family: $font-family-base
+    width: 300px
+    height: 560px
+    .phone
+        background-color: $brand-third
+        border-radius: 30px
+        border: 10px solid $white
+        padding: 10px 15px 0
+        margin-right: auto
+        margin-left: auto
+        nav.nav-container
+            display: flex
+            justify-content: space-between
+            padding: 20px 0 30px
+            .page-title
+                font-size: 0.6rem
+
+        section
+            background-color: $brand-light
+            border-radius: 2px 
+
+        .fas
+            font-family: 'FontAwesome' !important
```

Comment:
This could be moved to pseudo class `%font--awesome` or something like that and then you could use `@extend %font-awesome` just to not repeat it all over the place